### PR TITLE
Alpine build

### PR DESCRIPTION
**Pull request by az7arul**
Use alpine base image for ruby 2.5 to reduce size

### CODE REVIEW

File: 5.2-alpine/full/.dockerignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.ruby-version
+.git
+*.md
```
Comment (Adnilson) [inline 5.2-alpine/full/.dockerignore]: New line

File: 5.2-alpine/full/.gitignore [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/.bundle
+
+# Ignore all logfiles and tempfiles.
+/log/*
+/tmp/*
+!/log/.keep
+!/tmp/.keep
+
+.ruby-version
```
Comment (Adnilson) [inline 5.2-alpine/full/.gitignore]: new line

File: 5.2-alpine/full/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        mailcap \
+        && \
+    pip install --upgrade pip awscli==1.14.5 s3cmd==2.0.1 python-magic && \
+    apk -v --purge del py-pip && \
+    rm /var/cache/apk/*
+
+# RUN deps="curl awscli" \
+#  && apt-get update \
+#  && apt-get install -y --no-install-recommends $deps \
+#  && rm -rf /var/lib/apt/lists/*
```
Comment (bryszard) [inline 5.2-alpine/full/Dockerfile]: Why do we need this comment?

File: 5.2-alpine/full/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+#     DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
+#     C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
+#     B9AE9905FFD7803F25714661B63B535A4C206CA9 \
+#     56730D5401028683275BD23C23EFEFE93C4CFFFE \
+#     77984A986EBC2AA786BC0F66B01FBB92821C587A \
+#   ; do \
+#     gpg --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" || \
+#     gpg --keyserver hkp://ipv4.pool.sks-keyservers.net --recv-keys "$key" || \
+#     gpg --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" ; \
+#   done
```
Comment (bryszard) [inline 5.2-alpine/full/Dockerfile]: Do we need these comments?

File: 5.2-alpine/full/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+      tar -xf ${YARN_VERSION}.tar.gz -C /usr/local/share/yarn --strip 1 && \
+      ln -s /usr/local/share/yarn/bin/yarn /usr/local/bin/ && \
+      ln -s /usr/local/share/yarn/bin/yarnpkg /usr/local/bin/ && \
+      rm ${YARN_VERSION}.tar.gz*; \
+    fi; \
+  fi && \
+  apk del curl make gcc g++ python linux-headers binutils-gold gnupg ${DEL_PKGS} && \
+  rm -rf ${RM_DIRS} /node-${NODE_VERSION}* /usr/share/man /tmp/* /var/cache/apk/* \
+    /root/.npm /root/.node-gyp /root/.gnupg /usr/lib/node_modules/npm/man \
+    /usr/lib/node_modules/npm/doc /usr/lib/node_modules/npm/html /usr/lib/node_modules/npm/scripts
```
Comment (bryszard) [inline 5.2-alpine/full/Dockerfile]: That's a lot of commands :). You took it from somewhere or you wrote it?

File: 5.2-alpine/full/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+# Basic smoke test
+RUN node --version
+
+RUN mkdir -p /usr/src/rails
+WORKDIR /usr/src/rails
+
+COPY Gemfile /usr/src/rails/
+COPY Gemfile.lock /usr/src/rails/
+
+# nokogiri
```
Comment (bryszard) [inline 5.2-alpine/full/Dockerfile]: Nokogiri? I don't see it here.

File: 5.2-alpine/full/Dockerfile (hunk 5 above)
Comment (az7arul) [reply 5.2-alpine/full/Dockerfile]: copied from alpine node build

File: 5.2-alpine/full/Dockerfile (hunk 3 above)
Comment (az7arul) [reply 5.2-alpine/full/Dockerfile]: we don't, forgot to remove it

File: 5.2-alpine/full/Dockerfile (hunk 4 above)
Comment (az7arul) [reply 5.2-alpine/full/Dockerfile]: nope

**Pull request merged**